### DISCUSSION TITLE
Thank you for supporting Garmin watches!

### DISCUSSION THREAD

**Discussion by rctaylor157** [thrillfall/nextcloud-gpodder#97] [General]
Just stopping by to say thank you for supporting Garmin watches! I know it's a niche group of users, and I was actually shocked when I found out I could use this to keep things synced up with my watch too. It's awesome!

**Comment by irisv** [comment]
I'm going to adopt this as the thread for heaping on general praise...

So I'm a long time Nextcloud and AntennaPod user, and I love them both (though the gpodder sync errors were driving me a little crazy) --- and now they got married!!! I love this so, so much - thank you!
(I guess next up is getting a Garmin watch!)
